README.md

```diff
@@ -166,6 +166,9 @@ Each model represents the different kinds of requests and responses that can be
 
 ## Changelog
 
+1.7.0
+* New `getCustomerCardToken` method added to `CustomersAPI` for creating a card funding sources token for a customer.
```

Comment:
should _sources_ be singular on this part?

Reply:
The endpoint is called /card-funding-sources-token so I think that was what I was implying